scripts/ci-conformance.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
```

Comment:
A lot of this script was taken directly or inspired from https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/hack/ci/e2e-conformance.sh